### DISCUSSION TITLE
Are wildcards in the path supported?

### DISCUSSION THREAD

**Discussion by mathiasfritsch** [dotnet/yarp#692] [General] [answered]
Hi,

Are wildcards for the path supported? I tried /api/myendpoint* but that did not work for /api/myendpoint/getvalues

```
    "Routes": [
   {
    "RouteId": "routegreen",
    "ClusterId": "clustergreen",
    "Match": {
      "Path": "/api/myendpoint*"
    }
  },
```

greetings
Mathias

**Comment by Tratcher** [comment, marked as answer]
These become ASP.NET Core routes and use the same syntax. I think what you're looking for here is "/api/myendpoint/{**catch-all}".
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-5.0#route-template-reference
